docs/docker.md

```diff
@@ -19,6 +19,16 @@ specified in the `docker_image` field. This enables users to extend our base ima
 and run evaluations on an image that has all additional requirements for their defense.
 
 
+### Custom Images
+If you wish to utilize custom images for armory, these can be directly specified in
+either the `docker_image` field of the config file of `armory run <path/to/config.json>`
+or in the docker image field specified in `armory launch` or `armory exec`.
```

Comment:
What is the docker image field? Maybe write it as image CLI arg (with an example command)

Reply:
Updated.